tox.ini

```diff
@@ -44,6 +44,7 @@ description = adapter function testing using MotherDuck
 skip_install = True
 passenv = *
 commands = {envpython} -m pytest --profile=md --maxfail=2 {posargs} tests/functional/adapter
+base_python = 3.11
```

Comment:
I ran into [Error in pip install duckDB #9301](https://github.com/duckdb/duckdb/issues/9301), so had to explicitly set the Python version to 3.11 here to run the tests with `tox run -e md`

Reply:
It looks like I need to reverse this change for the CI to pass, see https://github.com/duckdb/dbt-duckdb/actions/runs/7720091362/job/21085376439